apps/staging-community/__tests__/cli

```diff
@@ -0,0 +1 @@
+../../staging-enterprise/__tests__/cli
```

Comment:
I've used symlinks so that `staging-community` runs the same tests as `staging-enterprise`